src/tokenized-line.coffee

```diff
@@ -183,6 +185,9 @@ class TokenizedLine
   buildSoftWrapIndentationDelta: ->
     _.reduce @softWrapIndentationTokens, ((acc, token) -> acc + token.screenDelta), 0
 
+  isEmpty: ->
+    @tokens.length == 0 || @hasOnlySoftWrapIndentation()
+
   hasOnlySoftWrapIndentation: ->
     @tokens.length == @softWrapIndentationTokens.length
```

Comment:
This line would cover also the empty case, but I found it more expressive to have a separate method (`isEmpty`) that expresses intent better.